### PR TITLE
client-cert: AD review feedback 

### PR DESCRIPTION
**Pull request by b---c**
AD review feedback (thread at https://lists.w3.org/Archives/Public/ietf-http-wg/2023JanMar/0088.html) and some minor editorial updates

### CODE REVIEW

File: draft-ietf-httpbis-client-cert-field.md [hunk 1]
```diff
@@ -323,14 +323,14 @@ from the TTRP). A TTRP MUST sanitize the incoming request before forwarding it
 on by removing or overwriting any existing instances of the fields. Otherwise,
 arbitrary clients can control the field values as seen and used by the backend
 server. It is important to note that neglecting to prevent field injection does
-not "fail safe" in that the nominal functionality will still work as expected
```
Comment (MikeBishop) [inline draft-ietf-httpbis-client-cert-field.md:326]: I suspect this was correct before; "fail safe" is a verb, where "fail-safe" would be an adjective or used as a noun.

File: draft-ietf-httpbis-client-cert-field.md [hunk 2]
```diff
@@ -323,14 +323,14 @@ from the TTRP). A TTRP MUST sanitize the incoming request before forwarding it
...
+not "fail-safe" in that the nominal functionality will still work as expected
 even when malicious actions are possible. As such, extra care is recommended in
 ensuring that proper field sanitation is in place.
 
 The communication between a TTRP and backend server needs to be secured against
 eavesdropping and modification by unintended parties.
 
-The configuration options and request sanitization are necessarily functionally
+The configuration options and request sanitization are necessary functionally
```
Comment (MikeBishop) [inline draft-ietf-httpbis-client-cert-field.md:333]:
```suggestion
The configuration options and request sanitization are necessary functionality
```

**Review by MikeBishop — APPROVED** [review]
Looks good; two nits.

File: draft-ietf-httpbis-client-cert-field.md [hunk 3]
```diff
@@ -323,14 +323,14 @@ from the TTRP). A TTRP MUST sanitize the incoming request before forwarding it
 on by removing or overwriting any existing instances of the fields. Otherwise,
 arbitrary clients can control the field values as seen and used by the backend
 server. It is important to note that neglecting to prevent field injection does
-not "fail safe" in that the nominal functionality will still work as expected
+not "fail-safe" in that the nominal functionality will still work as expected
```
Comment (b---c) [inline draft-ietf-httpbis-client-cert-field.md:326]:
```suggestion
not "fail safe" in that the nominal functionality will still work as expected
```

**Comment by b---c** [comment]
thanks Mike, nits have been addressed

**Pull request merged**